backend/src/apiserver/server/visualization_server.go

```diff
@@ -115,13 +113,17 @@ func (s *VisualizationServer) getVisualizationServiceURL(request *go_client.Crea
 	return s.serviceURL
 }
 
-func isVisualizationServiceAlive(serviceURL string) bool {
+func isVisualizationServiceAlive(serviceURL string) error {
 	resp, err := http.Get(serviceURL)
+
 	if err != nil {
-		glog.Error("Unable to verify visualization service is alive!", err)
-		return false
+		glog.Error("Unable to verify visualization service aliveness ", err)
+		return util.Wrap(err, fmt.Sprintf("Unable to verify visualization service aliveness by sending request to %s", serviceURL))
```

Comment:
nit: for reducing duplication

My suggestion was
```
myError := util.Wrap(err, fmt.Sprintf("Unable to verify visualization service aliveness by sending request to %s", serviceURL))
glog.Error(myError)
return myError
```

Reply:
Yeah I agree with your idea, fixed.